app/models/call_repository.rb

```diff
@@ -0,0 +1,25 @@
+class CallRepository
+  class NotFoundError < StandardError; end
+
+  # ch_id => champaign id
+  # ak_id => action_kit id
+  def self.set(ch_id, ak_id)
+    redis.set("call:#{ch_id}", ak_id)
+  end
+
+  def self.get(ch_id)
+    redis.get("call:#{ch_id}")
+  end
+
+  def self.get!(ch_id)
+    get(ch_id) || raise(NotFoundError.new "Can't find repository with id: #{ch_id}")
```

Comment:
Is repository an official Redis vocab word? if I found this in the logs I would assume it was trying to pull in something from github or something.